src/components/toolCard/index.tsx

```diff
@@ -60,7 +60,7 @@ export const ToolCard: React.FC<ToolCardProps> = ({
     };
 
```

Comment:
Not shown here but the relative date stuff from lines `45-50` and below is better handled by the RTL browser library which I have implemented in `src/utils/datetime/formatDaysAgo.ts`

Reply:
I think @christineNg1028 can handle this whenever she finishes the tool card PR